### PR TITLE
test(lint): Add tests for the "lint" script

### PR DESCRIPTION
**Pull request by nanovazquez**
This PR adds basic tests for the `lint` script.

It also refactors the code that tests the `stylelint` script to reuse some of the common functionality in both.

### CODE REVIEW

File: test/scripts/lint/lint.spec.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    beforeAll(() => {
+      args = ["lint", `--files=${MOCKS_FOLDER_PATH}/failing-file.ts`];
+      results = runExosScript(args);
+    });
+
+    it("should output errors", () => {
+      expect(results.stdout.includes(errorMessage)).toBe(true);
+    });
+
+    it("should exit with status code 0", () => {
```
Comment (LautaroPetaccio) [inline test/scripts/lint/lint.spec.ts:37]: This should not return 0.

File: test/scripts/stylelint/stylelint.spec.ts [hunk 2]
```diff
@@ -1,53 +1,41 @@
...
     });
 
-    it("should output stylelint errors", () => {
-      const hasErrorMessage = results.stdout.indexOf("❌ There were errors while running stylelint.") !== -1;
-      expect(hasErrorMessage).toBe(true);
+    it("should output errors", () => {
+      expect(results.stdout.includes(errorMessage)).toBe(true);
     });
 
     it("should exit with status code 0", () => {
```
Comment (LautaroPetaccio) [inline test/scripts/stylelint/stylelint.spec.ts:37]: Why do both runs, the positive and the negative ones return 0?

File: test/scripts/lint/lint.spec.ts (hunk 1 above)
Comment (nanovazquez) [reply test/scripts/lint/lint.spec.ts]: Fixed

File: test/scripts/stylelint/stylelint.spec.ts (hunk 2 above)
Comment (nanovazquez) [reply test/scripts/stylelint/stylelint.spec.ts]: Fixed

**Pull request merged**